package.json

```diff
@@ -16,7 +16,15 @@
     "test.ci": "jest --maxWorkers=2",
     "test.coverage": "yarn test --coverage",
     "test.watch": "yarn test --watchAll",
+    "download.prism.v2": "curl -L https://github.com/stoplightio/prism/releases/download/v2.0.17/prism_linux_amd64 -o prism && chmod +x ./prism",
```

Comment:
Okay, but why I shouldn't be able to run this locally? Why take this away from a dev? And even if you have prism locally, it doesn't matter, since here we are downloading to prism project root folder.

Reply:
I think that the scripts sections should really be stuff related to the real package development. 

For me (but that's a personal opinion) this is just distracting noise. If you really want these kind of scripts, I would create a `scripts.sh` file and put that stuff here. Not in `package.json` for sure. That's for Node, not for other things.

We can keep it here if you want, it's not a problem — but expect me to transfer these in the `.circleci/config` file once we testing harness goes in "maintenance" mode.

Reply:
I dont imagine that a developer will be doing this on a regular basis, or even ever. 

Asking why we shouldn't add code is backwards, we should ask why we should add code. Otherwise we get a kitchen sink. :)